build.gradle

```diff
@@ -6,14 +6,9 @@ plugins {
 group 'dev.ayu.matcha'
 version '1.0.0'
 
-test {
-    useJUnitPlatform()
-}
-
 application {
     mainClass.set("dev.ayu.matcha.Matcha")
     applicationDefaultJvmArgs = [
-            "-XX:MaxRAMPercentage=90.0",
             "-XX:+UseG1GC",
```

Comment:
Those JVM args were all copied from Tea; I've removed the 90% memory limit (since that can be better set at the container level, if required), but I'm not sure if the other options are all needed for Matcha, and what they even do for the most part.

Reply:
Rule of the thumb: A "professional" in StackOverflow says it works, so it must work! 😶😆